client/components/hunts.js

```diff
@@ -1,30 +1,35 @@
-import React from 'react';
-import { Text, TextInput, View, Button, StyleSheet } from 'react-native';
-import { connect } from 'react-redux';
+import React from 'react'
+import {Text, TextInput, View, Button, StyleSheet} from 'react-native'
+import {connect} from 'react-redux'
 //------------------------------------------------------------------
-import { fetchAllHunts } from '../store/hunts';
-import { fetchCreatedHuntLocations } from '../store/huntLocations';
+import {fetchAllHunts} from '../store/hunts'
+import {
+  fetchCreatedHuntLocations,
+  fetchDroppingHuntLocations
+} from '../store/huntLocations'
 //------------------------------------------------------------------
 // CONSTANTS
 //------------------------------------------------------------------
 class Hunts extends React.Component {
   constructor() {
-    super();
+    super()
   }
```

Comment:
Constructor does not need to be defined if not setting constructor props on component.